Formula/openjdk.rb

```diff
@@ -17,9 +16,10 @@ class Openjdk < Formula
 
   depends_on "autoconf" => :build
 
+  # From https://jdk.java.net/archive/
```

Comment:
Could we use the existing build for this?

Reply:
See https://github.com/Homebrew/homebrew-core/pull/45935#issuecomment-547801045